pandas/tests/frame/test_nonunique_indexes.py

```diff
@@ -452,3 +452,16 @@ def test_as_matrix_duplicates(self):
                             dtype=object)
 
         self.assertTrue(np.array_equal(result, expected))
+
+    def test_set_value_by_index(self):
+        # See gh-12344
+        df = pd.DataFrame({'Same': 1.0,
+                           ' Same': np.nan,
+                           '  Same': np.nan},
+                          index=[0, 1, 2])
+
+        df.columns = [c.strip() for c in df.columns]
```

Comment:
Done.  Would you mind also cancelling my first Travis build then (it hasn't started yet but fixing this test added a new build)?